StarkInfra/StarkInfra/CreditNote/CreditNote.cs

```diff
@@ -88,15 +89,15 @@ public partial class CreditNote : Resource
         /// <br/>
         /// Parameters (required):
         /// <list>
-        ///     <item>templateId [string]: ID of the contract template on which the credit note will be based. ex: templateId="0123456789101112"</item>
+        ///     <item>templateID [string]: ID of the contract template on which the credit note will be based. ex: templateID="0123456789101112"</item>
         ///     <item>name [string]: credit receiver's full name. ex: name="Edward Stark"</item>
-        ///     <item>taxId [string]: credit receiver's tax ID (CPF or CNPJ). ex: "20.018.183/0001-80"</item>
+        ///     <item>taxID [string]: credit receiver's tax ID (CPF or CNPJ). ex: "20.018.183/0001-80"</item>
         ///     <item>nominalAmount [integer]: amount in cents paid to the credit receiver, before deductions. ex: nominalAmount=11234 (= R$ 112.34)</item>
         ///     <item>scheduled [DateTime or string]: date of payment execution. ex: scheduled=datetime(2020, 3, 10)</item>
         ///     <item>invoices [list of Invoice objects]: list of Invoice objects to be created and sent to the credit receiver. ex: new List<Invoice>{ new Invoice(), new Invoice() }</item>
         ///     <item>payment [Payment object]: Payment object to be created and sent to the credit receiver. ex: payment=Payment()</item>
         ///     <item>signers [List of Signer objects]: signer's name, e-mail and delivery method for the contract. ex: signers= new List<Signer>{ new Signer(name: "Tony Stark", contact: "tony@starkindustries.com", method: "link")}</item>
```

Comment:
Tipo da variável **signers** deveria ser `List of CreditSigner objects`, não?

Reply:
A docstring deveria ser: `signer's name, contact and delivery method for the signature request`